seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/OrcReadStrategy.java

```diff
@@ -163,6 +167,73 @@ boolean checkFileType(String path) {
         }
     }
 
+    private SeaTunnelDataType<?> orcDataType2SeaTunnelDataType(TypeDescription typeDescription) {
+        switch (typeDescription.getCategory()) {
+            case BOOLEAN:
+                return BasicType.BOOLEAN_TYPE;
+            case INT:
+                return BasicType.INT_TYPE;
+            case BYTE:
+            case SHORT:
```

Comment:
Why not use `TINYINT`、`SMALLINT`?

Reply:
Becase these types in orc are saved in `LongVector`, so when we read these types using java api it is a `Long` object.